### DISCUSSION TITLE
Recurring high CPU utilization

### DISCUSSION THREAD

**Discussion by Stefomat** [nextcloud/all-in-one#1542] [Questions] [answered]
Hi all,

for the last three or four days, I have been experiencing the phenomenon that every few seconds, the CPU utilization of my Nextcloud AIO VM increases to 20-25%. At the same time, the VM's network throughput increases from almost 0 to 0.5 Mbps. After about 15 to 20 seconds, both settle down again and CPU utilization and network throughput drop back to 0. My Nextcloud has two active users and is installed as an AIO Docker container in a Debian 11 VM in Proxmox.

Does anyone have any ideas how I can get to the bottom of this behavior?
I would not be aware of anything major changing on my Nextcloud 3 or 4 days ago.

Greetings
Stefomat

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce this. Maybe it is one of your installed Nextcloud apps? 
However it is probably better to open a new thread on https://help.nextcloud.com regarding this as it is most likely unrelated to AIO directly.

**Reply by Stefomat** [reply]
Hi, sorry for the delay. After two days of this strange behavior, everything has calmed down again. If it happens again, I will report it there.
